Developer docs is not working/up-to-date

Looks like the recent commits removed `make pack` and now I follow the developer doc here: https://github.com/flanksource/karina/blob/master/docs/developer-guide/index.md but got this error when run `make`

```
➜ make     
go build -o ./.bin/karina -ldflags "-X \"main.version=v0.28.1-43-gbb64119-20210228002712\""  main.go
manifests/static.go:3:8: package embed is not in GOROOT (/usr/local/Cellar/go/1.15.7_1/libexec/src/embed)
pkg/platform/platform.go:8:2: package io/fs is not in GOROOT (/usr/local/Cellar/go/1.15.7_1/libexec/src/io/fs)
make: *** [build] Error 1
```

And this target `linux-static` is still referencing `pack` target: https://github.com/flanksource/karina/blob/master/Makefile#L37

--- a/Makefile
+++ b/Makefile
@@ -33,10 +33,6 @@ build:
 linux:
 	GOOS=linux go build -o ./.bin/$(NAME) -ldflags "-X \"main.version=$(VERSION)\""  main.go
 
-.PHONY: linux-static
-linux-static: pack
-	CGO_ENABLED=0 GOARCH=amd64 GOOS=linux go build -a -tags netgo -ldflags -w -o ./.bin/static/$(NAME) -ldflags "-X \"main.version=$(VERSION)\""  main.go
-
 .PHONY: darwin
 darwin:
 	GOOS=darwin go build -o ./.bin/$(NAME)_osx -ldflags "-X \"main.version=$(VERSION)\""  main.go

--- a/docs/developer-guide/index.md
+++ b/docs/developer-guide/index.md
@@ -38,3 +38,13 @@ make serve-docs
 Navigate to [http://localhost:8000](http://localhost:8000)
 
 Update the documentation sources located in the repository in `docs/` (and its subdirectories) and the mkdocs development server will live-reload the pages as soon as changed.
+
+## Common Issues
+
+* `make
+ go build -o ./.bin/karina -ldflags "-X \"main.version=v0.28.1-43-gbb64119-20210228002712\""  main.go
+ manifests/static.go:3:8: package embed is not in GOROOT 
+ pkg/platform/platform.go:8:2: package io/fs is not in GOROOT
+ make: *** [build] Error 1`
+ 
+ The version of go used to compile the application does not support the go embed directive.  Install Go version 1.16+ 

